index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Awesome Books Home Page</title>
+ <link rel="stylesheet" href="/Assets/css/styles.css">
+</head>
+<body>
+ <main class="main-section">
+  <h1 class="main-title">Awesome books</h1>
+  <ul class="book-list">
+
+  </ul>
+  <form class="add-book">
+   <div>
+    <input class="title" type="text" placeholder="title" aria-label="title">
+   </div>
+   <div>
+    <input class="author" type="text" placeholder="author" aria-label="author">
+   </div>
+   <button type="submit" class="add-btn">Add</button>
+  </form>
+ </main>
+ <script src="/Assets/js/app.js"></script>
```

Comment:
- [ ] Same thing happening here, I think you have connected the `js` with wrong path. (Hint: It should be `./Assets/js/app.js`). Because of this I won't able to see any effect of `Js`. Please fix that. :smile:

Reply:
Done